.github/dependabot.yml

```diff
@@ -9,3 +9,11 @@ updates:
     commit-message:
       prefix: meta
     open-pull-requests-limit: 10
+
+  - package-ecosystem: npm
+    directory: /tools/eslint
+    schedule:
+      interval: monthly
+    commit-message:
+      prefix: tools
+    open-pull-requests-limit: 10
```

Comment:
Can we ask dependabot to only open one pull request that updates all relevant packages? That would be less noisy and match what we have now with the deps updater.

Reply:
Yeah, updates can be grouped https://docs.github.com/en/code-security/dependabot/dependabot-version-updates/customizing-dependency-updates#grouping-dependabot-updates-into-one-pull-request. Can also group everything, except major versions

Reply:
Okay, they are grouped now :-). Anything else?